scripts/apps/search/directives/SearchParameters.js

```diff
@@ -28,7 +28,12 @@ export function SearchParameters($location, asset, tags, metadata, meta, common,
                 scope.query = params.q;
                 scope.flags = false;
                 scope.common = common;
-                scope.meta = meta = {};
+                scope.meta = meta;
+                for (var member in meta) {
```

Comment:
@gbbr checks are passing, so this P.R. actually fix the problem, dunno if there is a better option, but his one works :)